src/reducers/postEdits.js

```diff
@@ -122,7 +125,13 @@ export const tagDescriptionEdits = (state = {}, action) => {
   } else if (includes([START_POST_EDIT, CANCEL_POST_EDIT, CREATE_COMMENT], type)) {
     return {}
   } else if (type === EDIT_TAG_DESCRIPTION) {
-    return {...state, [payload.tag]: payload.description}
+    if (state.creating === true) {
+      return {creating: true, [payload.tag]: payload.description}
+    } else {
+      return {...state, [payload.tag]: payload.description}
+    }
+  } else if (type === CREATE_TAG_IN_POST_EDITOR) {
+    return {creating: true, '': ''}
```

Comment:
better to set up a different reducer for the creating toggle rather than shoehorn it in here. then you can drop the code in the component that looks for a special value and omits it